src/components/CommentAdder.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from "react";
+import { fetchArticleComments, postComment } from "../api";
+
+export default function CommentAdder({
+  article,
+  articleComments,
+  setArticleComments,
+}) {
+  const [comment, setComment] = useState({ author: "jessjelly", body: "" });
+  const [isCommentPosted, setIsCommentPosted] = useState(false);
+  const [error, setError] = useState(null);
+
+  const handleChange = (event) => {
+    const type = event.target.id
+    const input = event.target.value
+    setComment({...comment, [type]: input})
+    
+  }
+
+  const handleSubmit= (event) => {
+    event.preventDefault()
+    postComment(article.article_id, comment).then(() => {})
+    setIsCommentPosted(true)
+    setComment({ author: "jessjelly", body: "" })
+    setArticleComments([...articleComments, comment])
+  }
+  
+  return (
+    <>
```

Comment:
This fragment is unnecessary as it is only needed to wrap elements with no parents because you need to return "one thing" from a function